docs/reference.md

```diff
@@ -359,13 +364,30 @@ OpenWhisk has a few system limits, including how much memory an action uses and
 * A user can change the limit when creating the action.
 * A container cannot have more memory allocated than the limit.
 
-### Per namespace #Concurrent Invocation (#) (Default: 100)
+### Per action artifact (MB) (Fixed: 1MB)
+* The maximum size for the artifact (e.g., file name) containing action definition is 1 MB.
+* It's recommended for nodejs action to use a tool to concatenate all source code including dependencies into a single bundled file 
+
+### Per activation payload size (MB) (Fixed: 1MB)
+* Any activations (i.e. invoke action or fire a trigger) the payload can't be larger than 1MB
```

Comment:
The maximum `POST` content size for an action invokation or trigger firing is 1MB.